### PR TITLE
#2115 portuguese language support tested

### LINKED ISSUE #2115: Portugese language support
## Is your feature request related to a problem? Please describe.

We don't currently support portugese

## Describe the solution you'd like

To support portugese

## Implementation

This can probably be split over a few issues!

- Add a portugese language code, flag and add it to the modal selection for languages on the login page.
- Add the portugese code to all the `JSON` files
- export the localization files, add the translations and reimport!

## Describe alternatives you've considered

N/A

## Additional context

N/A

### PR DESCRIPTION
**Pull request by nmadruga**
Fixes #2115 

## Change summary

Just adding support to Portuguese language

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Check that "Português" can be selected in the languages menu
- Check that Portuguese strings are not being cut in:
  - [ ] Initial setup (to select which store to sync) window
  - [ ] Main window buttons
  - [ ] Sync window
  - [ ] Login window
  - [ ] Stocktakes list window
  - [ ] New Stocktake window
  - [ ] Customer Invoices list window
  - [ ] New Customer Invoice window
  - [ ] Customer Requisitions list window
  - [ ] New Customer Requisition window
  - [ ] Supplier Invoices list window
  - [ ] New Supplier Invoice window
  - [ ] Supplier Requisitions list window
  - [ ] New Supplier Requisition window
  - [ ] Dashboard(?)
  - [ ] Dispensary patients list window
  - [ ] Dispensary prescriptions list window
  - [ ] Dispensary New patients window
  - [ ] Dispensary New prescription window
  - [ ] Any other places?

### Related areas to think about

Not happy about how the `Supplier Requisition` is showing, but I have tried for 15 minutes and think someone else should look. The component is the MenuButton and the styles are set here: https://github.com/openmsupply/mobile/blob/master/src/globalStyles/buttonStyles.js

### CODE REVIEW

File: src/localization/navStrings.json [hunk 1]
```diff
@@ -92,5 +92,27 @@
...
+    "invoice": "Fatura",
+    "language": "LÍNGUA",
+    "log_out": "DESCONECTAR",
+    "manage_stocktake": "Gerenciar inventários",
+    "new_stocktake": "Novo inventário",
+    "requisition": "Requisição",
+    "stocktakes": "Inventários",
+    "stocktake": "Inventário",
+    "supplier_invoices": "Faturas de fornecedores",
+    "supplier_requisitions": "Requisições de fornecedores",
```
Comment (joshxg) [inline src/localization/navStrings.json]: can just do `  "supplier_requisitions": "Requisições de\nfornecedores",` I think - I can't see a text wrapping prop !

Comment (nmadruga) [reply src/localization/navStrings.json]: Ah! Good idea. I'll try that now.

Comment (nmadruga) [reply src/localization/navStrings.json]: Yep. That worked Thanks!

**Comment by nmadruga** [comment]
I reviewed the Strings again and decided that it's clearer in Portuguese to have to/from included in the Requisitions and Invoices types:

**Comment by joshxg** [comment]
hmm sim, eu também acho

**Comment by joshxg** [comment]
você pode solicitar novamente a revisão quando terminar de revisar as strings? :)

**Comment by nmadruga** [comment]
@joshxg @Rebecca367 OK. Now I think this PR is complete to be reviewed! 🍾

**Comment by joshxg** [comment]
@nmadruga will try and look at it today or tomorrow but need to get the `4.0.0` release out and still need to do a lot of testing.

I am not sure if it needs to be in the `4.0.0` release? Could it be in a patch afterward? like, the next day ?

**Comment by nmadruga** [comment]
I don't see why not. Focus on the big release - way more important : )

**Pull request merged**